doc/user_manual/postprocessor.tex

```diff
@@ -1089,6 +1089,75 @@ \subsubsection{Interfaced}
                               'outputVar1':array([ .14,.15,.16])}
 \end{lstlisting}
 
+
+\paragraph{Method: HStoPSOperator}
+
+This Post-Processor performs the conversion from HistorySet to PointSet performing a projection of the output space.
+
+In the \xmlNode{PostProcessor} input block, the following XML sub-nodes are available:
+
+\begin{itemize}
+   \item \xmlNode{pivotParameter}, \xmlDesc{string, optional field}, ID of the temporal variable. Default is ``time''. 
+   \nb Used just in case the  \xmlNode{pivotValue}-based operation  is requested
+    \item \xmlNode{operator}, \xmlDesc{string, optional field}, the operation to perform on the output space:
+      \begin{itemize}
+        \item \textbf{min}, compute the minimum along each single history
+         \item \textbf{max}, compute the maximum along each single history
+         \item \textbf{min}, compute the average along each single history
```

Comment:
For my clarification, does this find the minimum of the pivot and then take that row?  Or does it get the minimum for each variable?  If I recall correctly we have both features in the HistorySetSnapShot InterfacedPostProcessor.

Reply:
This takes the maximum or min or average of each variable independently on the pivotParameter.

I checked the HisorySetSnapShot.... there is some overlapping between these two PP. We should merge them. I create an issue (#569)